SPECS/xcp-ng-release.spec

```diff
@@ -570,7 +570,8 @@ if [ -f /etc/systemd/system/chrony-wait.service ]; then
 fi
 
 # XCP-ng: Enable chronyd and chrony-wait services.
-#         They are not active in case of yum update from 8.0.
+#         They are not active in case of yum update from 8.0
+# In 8.2.1 they're still not active. We need to manually enabled them.
```

Comment:
```suggestion
#         They are not active in case of yum update from 8.0 because they're disabled in the preset file:
#         /usr/lib/systemd/system-preset/89-default.preset
#         An alternative fix would be to fix or patch the preset file.
```